keel-sql/src/main/resources/db/changelog/20200312-fix-timestamps.yml

```diff
@@ -0,0 +1,63 @@
+databaseChangeLog:
+  - changeSet:
+      id: fix-timestamps
+      author: lpollo
+      changes:
+        - modifyDataType:
+            tableName: delivery_config_last_checked
+            columnName: at
+            newDataType: timestamp(3)
+        - modifyDataType:
+            tableName: environment_artifact_constraint
+            columnName: created_at
+            newDataType: timestamp(3)
+        - sql:
+            sql: >
+              alter table keel.environment_artifact_constraint
+              modify judged_at timestamp(3) NULL default NULL
+        - modifyDataType:
+            tableName: environment_artifact_versions
+            columnName: approved_at
+            newDataType: timestamp(3)
+        - sql:
+            sql: >
+              alter table keel.environment_artifact_versions
+              modify deployed_at timestamp(3) NULL default NULL
```

Comment:
Some of these had to be written in SQL due to this error:
```
> liquibase.exception.MigrationFailedException: Migration failed for change set /Users/lpollo/netflix/spinnaker/workspace/keel/keel-sql/src/main/resources/db/changelog/20200312-fix-timestamps.yml::fix-timestamps::lpollo:
       Reason: liquibase.exception.DatabaseException: Invalid default value for 'deployed_at' [Failed SQL: (1067) ALTER TABLE keel.environment_artifact_versions MODIFY deployed_at timestamp(3)]
```

Reply:
Another way to handle that would be to use Liquibase's `dropNotNullConstraint` but this is fine too.